src/SDKs/CognitiveServices/dataPlane/Vision/FormRecognizer/FormRecognizer.Tests/Microsoft.Azure.CognitiveServices.Vision.FormRecognizer.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <Description>Microsoft.Azure.CognitiveServices.Vision.FormRecognizer.Tests Class Library</Description>
+    <AssemblyName>Microsoft.Azure.CognitiveServices.Vision.FormRecognizer.Tests</AssemblyName>
+    <Version>1.0.0</Version>    
```

Comment:
We generally recommend starting versioning from `0.9.0-preview` for a brand new SDK